foundation/org.eclipse.persistence.core/src/org/eclipse/persistence/config/PersistenceUnitProperties.java

```diff
@@ -3798,6 +3798,19 @@
      */
     public static final String USE_LOCAL_TIMESTAMP = "eclipselink.locking.timestamp.local." + PersistenceUnitProperties.DEFAULT;
 
+    /**
+     * The "<code>eclipselink.jpa.sqlcall.deferral.default</code>" property defines if SQL calls should be deferred to end of
+     * transaction by default or not. When setting this property to <code>false</code>, the application assumes the responsibility 
+     * of ordering the SQL statements and must therefore be aware of any interdependencies between entities. 
+     * <p>
+     * <b>Allowed Values</b> (case sensitive String)<b>:</b>
```

Comment:
the code in entitymanagersetupimpl does case in-sensitive comparison, so it does not match what is said here - you wanted to say `case insensitive String` here perhaps?

Reply:
Yep, thx for catching this.